### ISSUE TITLE
"Configuration for rule "compat/compat" is invalid"

### ISSUE THREAD

**Issue by stanlrt** [amilajack/eslint-plugin-compat#639]
I am using the following config (plain `index.js` file, using ESLint 8) inside of the `rules` object:
```typescript
"compat/compat": [
  "error",
  {
    ignore: ["op_mini all"],
  },
],
``` 

Very weirdly, the linting itself works fine (errors are ignored appropriately), however the extension and the pre-commit hooks complain that:
```text
Configuration for rule "compat/compat" is invalid:
	Value {"ignore":["op_mini all"]} should be string.
``` 

Following the recommendation breaks the linting.

**Comment by stanlrt** [comment]
```typescript
"compat/compat": [
			"error",
			"not op_mini all",
		],
```
